### ISSUE TITLE
Enable launcher on Android 13

### ISSUE THREAD

**Issue by nya-elimu** [elimu-ai/launcher#69]
Make it possible to set the elimu.ai Launcher as the default launcher app in the Android settings.

The current version of the app does not appear in the list of launcher apps on Android 13:

![Screenshot_2023-05-10-11-39-07-06_651b5c74a671b75cef00c5b91170e3c1.jpg](https://github.com/elimu-ai/launcher/assets/15718174/4f8fb450-3c4b-4afe-b985-d23c82fa1d90)

**Comment by tuancoltech** [comment]
@jo-elimu Are you able to recreate this issue?

**Comment by jo-elimu** [comment]
@tuancoltech Yes, seems like this is only a problem on devices with ColorOS. On devices running plain Android, the setting appears:

## Android 14 (API 34) - ColorOS

![Image](https://github.com/user-attachments/assets/ace30c8f-3d66-496a-b814-b2ae48874b0a)

![Image](https://github.com/user-attachments/assets/e0ee708e-094f-4cb8-b920-659ed4ed8eeb)

## Android 12L (API 32)

![Image](https://github.com/user-attachments/assets/87639a5a-297c-4d7e-af08-4f2999f38bd5)

**Comment by jo-elimu** [comment]
Anyway; Once https://github.com/elimu-ai/launcher/issues/82 gets fixed, this problem becomes less relevant. Because people with devices that allow this setting, can just launch it manually. It's not the best option, but at least it enables a way to access the software. So we can close this issue for now.
